src/math/geometry.cpp

```diff
@@ -243,6 +243,81 @@ int containedInTriangle(
   return NOT_CONTAINED;
 }
 
+ConvexityResult isConvexQuad(std::array<Eigen::VectorXd, 4> coords)
```

Comment:
coords could now be `const`, right?

Reply:
I prefer to pass them by value here, as we need to transform them internally. So this just saves us to explicitly copy them inside the function.
If that was not the case `const T&` would probably make sense.